### PR TITLE
Adding `_github-pages-challenge-cghs-hackclub.cghs.hackclub.com`

### PR DESCRIPTION
**Pull request by Codaea**
# Updating `cghs.hackclub.com`

## Description

added TXT record for GitHub pages challenge

### CODE REVIEW

**Review by reesericci — CHANGES_REQUESTED** [review]
Invalid YAML (line 170)

File: hackclub.com.yaml [hunk 1]
```diff
@@ -166,6 +166,10 @@ _github-challenge-rahs-hackclub.rahs:
   - ttl: 1
     type: TXT
     value: f43a54836a
+_github-pages-challenge-cghs-hackclub.cghs:
+  -ttl: 1
+    type: TXT
```
Comment (Codaea) [inline hackclub.com.yaml:171]: missing space
```suggestion
  - ttl: 1
```

File: hackclub.com.yaml [hunk 2]
```diff
@@ -166,6 +166,10 @@ _github-challenge-rahs-hackclub.rahs:
   - ttl: 1
     type: TXT
     value: f43a54836a
+_github-pages-challenge-cghs-hackclub.cghs:
+  - ttl: 1
```
Comment (reesericci) [inline hackclub.com.yaml:170]: TTL should be at least 600, 3600 is prolly good

**Review by reesericci — CHANGES_REQUESTED** [review]

**Review by Codaea — COMMENTED** [review]
update TTL to 3600

**Review by Muirrum — CHANGES_REQUESTED** [review]
It looks like you've accidentally committed a merge conflict without resolving it

File: hackclub.com.yaml [hunk 3]
```diff
@@ -166,6 +166,14 @@ _github-challenge-rahs-hackclub.rahs:
...
     type: TXT
     value: f43a54836a
+_github-pages-challenge-cghs-hackclub.cghs:
+<<<<<<< HEAD
+  - ttl: 7200
+=======
+  - ttl: 1
+>>>>>>> 3a8deff03edc89d6d40b043befac311b82ceb563
+    type: TXT
+    value: a1704fb19aca2f4b785803f2f2edb8
```
Comment (reesericci) [inline hackclub.com.yaml:176]: I think you forgot to resolve merge conflicts before pushing

**Comment by Codaea** [comment]
GitHub doin some funky stuff, hold on

**Review by Muirrum — APPROVED** [review]
lgtm

**Review by reesericci — APPROVED** [review]
lgtm

**Pull request merged**